config/feature.js

```diff
@@ -23,6 +23,10 @@ function isFirefoxPanel() {
   return DebuggerTarget == "firefox-proxy";
 }
 
+function isFirefox() {
+  return /firefox/i.test(navigator.userAgent)
```

Comment:
Are you sure this won't have any false positives? I feel like I've seen checks that are a lot more complicated than this.